lib/improver/cli/__init__.py

```diff
@@ -220,6 +220,36 @@ def with_intermediate_output(wrapped, *args, intermediate_output=None,
     return result
 
 
+def create_constrained_inputcube_converter(*constraints):
+    """Makes a list of functions with different constraints.
+
+    Args:
+        *constraints (string):
+            constraints to be used in the loading of cubes against a cubeList
+
+    Returns:
+        list of function:
+            A list of functions with different constraints.
+    """
+    @value_converter
+    def constrained_inputcube_converter(to_convert):
+        """Passes the cube and constraints onto maybe coerce with.
+
+        Args (iris.cube.Cube):
+            To cube to be passed forward for returning or loading.
+
+        Returns:
+            iris.cube.CubeList:
+                The loaded cubelist of constrained cubes.
+        """
+        from improver.utilities.load import load_cube
+        import iris
```

Comment:
If we only need `iris.cube.CubeList` , can we just `import` this please?

Reply:
Done